recipes/lunasvg/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,8 @@
-cmake_minimum_required(VERSION 3.1)
-project(test_package)
-
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(TARGETS)
+cmake_minimum_required(VERSION 3.8)
+project(test_package CXX)
 
 find_package(lunasvg REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
-target_link_libraries(${PROJECT_NAME} lunasvg::lunasvg)
-set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 14)
+target_link_libraries(${PROJECT_NAME} PRIVATE lunasvg::lunasvg)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
```

Comment:
not 17 for new version?

Reply:
Probably it does not expose C++17 features on public headers.